src/userlist/UsersCard.js

```diff
@@ -32,6 +34,10 @@ export default class UsersCard extends Component {
     filter: string,
   };
 
+  static contextTypes = {
+    drawer: () => null,
```

Comment:
We probably want to make the drawer's open/close status part of the app's state eventually